src/views/dmca/dmca.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../colors";
+@import "../../typography";
```

Comment:
I'm a bit torn on this – on one hand, doing the import here and then importing the `dmca.scss` file in `dmca.jsx` keeps to our conventions, however it seems that if all we're doing is importing in a view's scss file that we could just as easily do those imports on the jsx file directly, and delete the scss file.